src/app/(simulation)/(large-layout)/organisations/creer/_components/CreationForm.tsx

```diff
@@ -140,30 +133,6 @@ export default function CreationForm() {
               </option>
             ))}
           </Select>
-
-          {watch('organisationType') ===
-            OrganisationTypeEnum.groupOfFriends && (
-            <div className="mt-4 rounded-xl bg-gray-100 p-4 text-sm">
-              <p className="mb-2">
-                <Trans>
-                  Le mode organisation est un mode <strong>100% anonyme</strong>{' '}
-                  pour les participants.
-                </Trans>
-              </p>
-
-              <p className="mb-4">
-                <Trans>
-                  Avez-vous essayé{' '}
-                  <strong>notre fonctionnalité “Groupes d’amis”</strong> ? Elle
-                  vous permettra de vous comparer dans un classement : que celui
-                  ou celle ayant la plus faible empreinte gagne !
-                </Trans>
-              </p>
-              <ButtonLink href={linkToGroupCreation} size="sm">
-                <Trans>Créer un groupe d'amis</Trans>
-              </ButtonLink>
-            </div>
-          )}
```

Comment:
Why remove this ? Not in the card

Reply:
okay moving that to another card / branch